mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": 0,
+            "available": True,
+            "status": "success",
+            "metrics": {
+                "user_percent_average": {
+                    "data": 7.1695253641707,
```

Comment:
TBH, and as an Intelligence dev I feel like this is a good use case for the backplane. For now while we're still developing the backplane, we should be intentional about the values we return. After all, these are mocked metrics, not some mystical constants handed down by the programming gods. The last thing I would want to do as an Intelligence dev is do something forward incompatible such that Mimic is stuck with `7.1695253641707` forever.

For now, I feel being intentional about our output means not going crazy with decimal places (max 1 after the decimal point is great!) and leaving clear documentation to the effect that Mimic may change these values at any time without warning.